lib/dry/core/class_attributes.rb

```diff
@@ -1,16 +1,11 @@
 # frozen_string_literal: true
 
-require "dry/core/constants"
-require "dry/core/errors"
-
 module Dry
   module Core
     # Internal support module for class-level settings
     #
     # @api public
     module ClassAttributes
-      include Constants
```

Comment:
But this should have broken tests :)

Reply:
@flash-gordon I included `Constants` in `Dry::Core`

Reply:
Ah OK